src/widget/form/settings/generalform.cpp

```diff
@@ -339,5 +337,14 @@ void GeneralForm::on_checkUpdates_stateChanged()
  */
 void GeneralForm::retranslateUi()
 {
+    auto createLink = [this](QString path, QString text) {
+        return QStringLiteral(
+                   "<a href=\"%1\" style=\"text-decoration: underline; color:%2;\">%3</a>")
+            .arg(path, style.getColor(Style::ColorPalette::Link).name(), text);
+    };
```

Comment:
why the callable? seems like extra steps here.

Reply:
Ok, moved it out.